src/db_impl_gc.cc

```diff
@@ -81,6 +81,14 @@ Status TitanDBImpl::BackgroundGC(LogBuffer* log_buffer) {
         assert(column_family_id == cfh->GetID());
         blob_gc->SetColumnFamily(cfh.get());
       }
+
+      if (blob_gc->trigger_next()) {
```

Comment:
to avoid overwhelming the queue, maybe not trigger next if there are already a lot of jobs to do?